PrivateCollections/Germany/Neukirchen006.xml

```diff
@@ -356,148 +363,82 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                        <locus target="#86r"></locus>
-                        <desc>Directive as to how to use Ps. 91,written
-                          in small fine script, underlined.</desc>
-                        <q xml:lang="amh">ይኄይስ᎓ ተአምኖ᎓ እስከ᎓ ስምዓኒ᎓ ድረስ᎓ ድግም᎓
-                        መፍትሔ᎓ ሥራይ᎓ ተሐጸብ᎓ ኵለንታከ᎓ ፯ጊዕለት᎓</q>
-                      </item>
-
-                      <item xml:id="e11">
-                        <locus target="#98r"></locus>
-                        <desc type="Unclear">Numerals added near Ps. 105:25,
-                          possibly in the main hand.</desc>
-                        <q xml:lang="amh">፳፺፬፩</q>
-                      </item>
-
-                      <item xml:id="e12">
-                        <locus from="3r" to="159v"></locus>
-                        <desc>The use of compressed script to accomodate longer lines,
-                        corrections, take-downs and take-ups of the lines on some folios.</desc>
-                      </item>
-
-                      <item xml:id="e13">
-                        <locus from="3r" to="159v"></locus>
-                        <desc>Scribbles in crude hands on a few folios.</desc>
+                        <locus from="3r" to="127v"></locus>
+                        <desc>Scribbles with pencil, on a few folios.</desc>
                       </item>
                      </list>
                   </additions>
 
                   <bindingDesc>
                      <binding contemporary="true" xml:id="binding">
-                        <decoNote xml:id="b1" type="Boards">Two wooden boards of slightly irregular size,
-                          cut not exactly along the grain but somewhat diagonally,
-                          discolored, with traces of adhesives on the inner sides.</decoNote>
-                        <decoNote xml:id="b2" type="Cover">Untooled leather cover
-                          (quarter cover) that was glued to the boards (still holding on the
-                          back board); turn-ins are held by threads.</decoNote>
-                        <decoNote xml:id="b3" type="SewingStations">4 sewing stations. The sewing is damaged,
-                          loose. The quires are held together by means of thick threads led
-                          through the first five quires, and through the last quires and the back board
-                          (of animal origin), and knoted on the spine.</decoNote>
-                          <decoNote xml:id="b4" type="EndLeaves"><locus from="1r" to="2v"/> are made from
-                        reused leaves with text, originating from a different, more recent manuscript.</decoNote>
-                          <decoNote xml:id="b5" type="bindingMaterial">
+                        <decoNote xml:id="b1" type="Boards">Two wooden boards.</decoNote>
+                        <decoNote xml:id="b2" type="SewingStations">4 sewing stations, the sewing intact.
```

Comment:
this type only takes a numeral value; the text must go in type="Other"

Reply:
Done.